test/setup.sh

```diff
@@ -5,14 +5,23 @@ UPTEST_GCP_PROJECT=${UPTEST_GCP_PROJECT:-crossplane-playground}
 
 echo "Running setup.sh"
 echo "Waiting until configuration package is healthy/installed..."
-${KUBECTL} wait configuration.pkg platform-ref-gcp --for=condition=Healthy --timeout 5m
-${KUBECTL} wait configuration.pkg platform-ref-gcp --for=condition=Installed --timeout 5m
+${KUBECTL} wait configuration.pkg platform-ref-gcp --for=condition=healthy --timeout 5m
```

Comment:
why do we need to switch cases here?

Just quickly checked status yaml of one of installed Configurations:
```
status:
  conditions:
  - lastTransitionTime: "2022-10-17T08:37:38Z"
    reason: HealthyPackageRevision
    status: "True"
    type: Healthy
```

Reply:
No special reason, noticed both works and wanted to go with lowercase consistently. 
From help

```
    --for='':
	The condition to wait on: [delete|condition=condition-name[=condition-value]|jsonpath='{JSONPath
	expression}'=JSONPath Condition]. The default condition-value is true.  Condition values are compared after
	Unicode simple case folding, which is a more general form of case-insensitivity.
```

Reply:
Reverted not to lead confusion though.